backend/api/Controllers/MissionSchedulingController.cs

```diff
@@ -361,6 +360,18 @@ [.. missionInspectionAreaNames]
                 return NotFound($"No area found for mission '{missionDefinition.Name}'.");
             }
 
+            if (
+                !inspectionAreaService.MissionTasksAreInsideInspectionAreaPolygon(
+                    missionTasks,
+                    area.InspectionArea
+                )
+            )
+            {
+                return Conflict(
+                    $"The tasks of the mission are not inside the inspection area of the robot"
+                );
```

Comment:
Is conflict correct error type or is bad request more descriptive? Conflict indicates that the request didn't success at the moment due to some conflict with the current state, but this request will never succeed with the given input.

Reply:
Yeah, I agree, Ill change it to BadRequest